src/commands/set.js

```diff
@@ -96,12 +96,15 @@ const setBoolean = (dotNotation, value) => {
 	return attemptWriteToFile(value, dotNotation);
 };
 
-const setString = (dotNotation, value) => {
+const setValue = (dotNotation, value) => {
 	const dotNotationArray = dotNotation.split('.');
 	dotNotationArray.reduce(setNestedConfigProperty(value), config);
 	return attemptWriteToFile(value, dotNotation);
```

Comment:
The message in `attemptWriteToFile` needs to handle arrays. Currently you get this for an empty array:
```
{
	"message": "Successfully set api.nodes to ."
}
```

Reply:
how about update to
`Successfully reset api.nodes.`